src/parsers/i3d/loadCompressedGeometryData.ts

```diff
@@ -9,7 +9,12 @@ export default function loadGeometryIndices(file: CustomFileReader, sectorEndLoc
 
   const geometryIndices: CompressedGeometryData[] = [];
   while (file.location < sectorEndLocation) {
-    const type = IdToFileGeometryName[file.readUint8()];
+    const typeId = file.readUint8();
+    const type = IdToFileGeometryName[typeId];
+    if (type === undefined) {
+      // This will happen if file type 4 is used
```

Comment:
What's the deal with this?

Reply:
You'll notice that I have a bunch of console.warns everywhere. They basically yell at you if the parser parameters are wrong.

Reply:
But long story with file versions. I'll take this out once I make file type 5 the default.

Reply:
Could you make a clear TODO here (so it's searchable)